### PR TITLE
team 2 frontend

### PR DESCRIPTION
**Pull request by SMJD11**

### CODE REVIEW

File: README.md
Comment (ECMGit) [inline README.md:1]: Don't touch my README.md

File: src/pages/user/Profile.tsx
Comment (ECMGit) [inline src/pages/user/Profile.tsx:1]: don't change this file. Or you can merge latest update from my repo to keep this content on.

**Review by ECMGit — CHANGES_REQUESTED** [review]

File: src/pages/user/Profile.tsx
Comment (SMJD11) [reply src/pages/user/Profile.tsx:1]: Updated to match original fork. Please let me know next steps

File: README.md
Comment (SMJD11) [reply README.md:1]: Updated to match original fork. Please let me know next steps

Comment (ECMGit) [reply README.md:1]: send me new PR please

**Pull request closed without merging**